presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -231,16 +257,27 @@ public boolean isDecomposable(FunctionRegistry functionRegistry)
         private final FunctionCall call;
         private final Signature signature;
         private final Optional<Symbol> mask;
+        private final List<Symbol> orderBy;
+        private final List<SortOrder> ordering;
```

Comment:
I think it would be nice to extract changes in `AggregationNode` as separate commit, that way it will split simple refactoring from real changes in planner